db/migrate/20190712072221_create_entries_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEntriesTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :entries do |t|
+    	t.string :phrase
+    	t.string :pinyin
+    	t.integer :view_count
```

Comment:
do we need to worry about decimal view_count? this will only allow for integers :)